util/pbtypes/copy.go

```diff
@@ -30,7 +30,7 @@ func CopyBlock(in *model.Block) (out *model.Block) {
 	return
 }
 
-func CopyStruct(s *types.Struct) *types.Struct {
+func CopyStruct(s *types.Struct, copyVals bool) *types.Struct {
```

Comment:
would be nice to have a test for the flag

Reply:
@fat-fellow added some tests